benefits/core/migrations/0001_initial.py

```diff
@@ -103,13 +118,62 @@ class Migration(migrations.Migration):
                 ("jws_signing_alg", models.TextField(help_text="The JWS-compatible signing algorithm.")),
-                ("payment_processor", models.ForeignKey(on_delete=models.deletion.PROTECT, to="core.paymentprocessor")),
                 ("eligibility_types", models.ManyToManyField(to="core.EligibilityType")),
-                ("eligibility_verifiers", models.ManyToManyField(to="core.eligibilityverifier")),
-                # fmt: on
+                ("eligibility_verifiers", models.ManyToManyField(to="core.EligibilityVerifier")),
+                (
+                    "payment_processor",
+                    models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to="core.paymentprocessor"),
+                ),
+                (
+                    "private_key",
+                    models.ForeignKey(
+                        help_text="The Agency's private key, used to sign tokens created on behalf of this Agency.",
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name="+",
+                        to="core.pemdata",
+                    ),
+                ),
             ],
         ),
+        migrations.AddField(
+            model_name="paymentprocessor",
+            name="client_cert",
+            field=models.ForeignKey(
+                help_text="The certificate used for client certificate authentication to the API.",
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="+",
+                to="core.pemdata",
+            ),
+        ),
+        migrations.AddField(
+            model_name="paymentprocessor",
+            name="client_cert_private_key",
+            field=models.ForeignKey(
+                help_text="The private key, used to sign the certificate.",
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="+",
+                to="core.pemdata",
+            ),
+        ),
+        migrations.AddField(
+            model_name="paymentprocessor",
+            name="client_cert_root_ca",
+            field=models.ForeignKey(
+                help_text="The root CA bundle, used to verify the server.",
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="+",
+                to="core.pemdata",
+            ),
+        ),
+        migrations.AddField(
+            model_name="eligibilityverifier",
+            name="public_key",
+            field=models.ForeignKey(
+                help_text="The Verifier's public key, used to encrypt requests targeted at this Verifier and to verify signed responses from this verifier.",  # noqa
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="+",
+                to="core.pemdata",
+            ),
+        ),
```

Comment:
The changes to this file look fine to me. It does make me curious though: what steps did you follow to get this result, and should that be our norm be for updating this file? 

(Based on the comment at the top that this was generated, I would guess the steps were to delete the file and then run the `manage.py` module with something like `makemigrations`?)

Reply:
Ah thanks for including that -- I should've checked the commit messages. Re: writing docs on this, I'd be ok with it being part of this PR or as follow up. Either way works for me!

Reply:
Took a follow-up in #683